audit.json

```diff
@@ -0,0 +1,6 @@
+{
+  "100001_Unexpected Content-Type was returned_http://am-role-assignment-service-aat.service.core-compute-aat.internal/v2/api-docs_GET": "ignore",
+  "10038_Content Security Policy (CSP) Header Not Set_http://am-role-assignment-service-aat.service.core-compute-aat.internal/v2/api-docs_GET": "ignore",
+  "100000_A Server Error response code was returned by the server_http://am-role-assignment-service-aat.service.core-compute-aat.internal/v2/api-docs_GET": "ignore",
+  "10036_Server Leaks Version Information via \"Server\" HTTP Response Header Field_http://am-role-assignment-service-aat.service.core-compute-aat.internal/v2/api-docs_GET":  "ignore"
+}
```

Comment:
Should these be ignored? Or can they be resolved by us?

Reply:
I think, we can check about these issues later.